generators/client/templates/angular/src/main/webapp/app/shared/util/datepicker-adapter.ts.ejs

```diff
@@ -30,10 +30,12 @@ export class NgbDateMomentAdapter extends NgbDateAdapter<Moment> {
         if (date != null && moment.isMoment(date) && date.isValid()) {
             return { year: date.year(), month: date.month() + 1, day: date.date() };
         }
-        return null;
+        // ! can be removed after https://github.com/ng-bootstrap/ng-bootstrap/issues/1544 is resolved
+        return null!;
     }
 
     toModel(date: NgbDateStruct): Moment {
-        return date ? moment(date.year + '-' + date.month + '-' + date.day, 'YYYY-MM-DD') : null;
+        // ! can be removed after https://github.com/ng-bootstrap/ng-bootstrap/issues/1544 is resolved
```

Comment:
I don't understand the comment, I don't see any `!`

Reply:
This is at the end after `null`, I modified comment to point out that this is after `null`